src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/config/ResourceServerConfiguration.kt

```diff
@@ -27,6 +27,7 @@ class ResourceServerConfiguration {
           "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html",
           "/queue-admin/retry-all-dlqs",
           "/incentives/reports/reconciliation",
+          "/appointments", "/activities",
```

Comment:
You've got a role on the endpoints so we don't need to bypass security here? I don't think we're going to have a cronjob to perform the deletes are we?

Reply:
whoops good spot! will remove !!